src/openrct2/drawing/Drawing.h

```diff
@@ -17,6 +17,7 @@
 #include <optional>
 #include <vector>
 struct ScreenCoordsXY;
+struct ScreenRect;
```

Comment:
```suggestion
```

`ScreenRect` is already forward declared 4 lines below. While we're at it, let's also remove this duplicate of forward declaration for `ScreenCoordsXY`